app/assets/javascripts/admin_legacy.js

```diff
@@ -0,0 +1,14 @@
+//= require govuk_publishing_components/lib
+//= require jquery-ui
+//= require extensions_legacy/jquery.distinct
+//= require vendor_legacy/jquery/chosen.jquery.js
+//= require vendor_legacy/jquery/jquery.player.min.js
+//= require vendor_legacy/jquery/jquery.highlighttextarea.js
+//= require vendor_legacy/jquery/jquery-debounce.js
+//= require vendor_legacy/google-diff-match-patch.js
+//= require ./application_legacy/convert-html-pub-charts
+//= require govuk_legacy
+//= require_tree ./common_legacy
```

Comment:
Similar comment about this only being legacy in admin context, but by it's nature of being common it's things for frontend and backend.

I think we should leave common as is. If we wanted to distinguish we could copy files from common into admin_legacy

Reply:
I think maybe we should have two copies, one in `admin_legacy` and one in `frontend`. Leaving it in the root folder might indicate that these things are actually common